### PR TITLE
Update Panel

### PR DESCRIPTION
**Pull request by saralohr**

### CODE REVIEW

File: src/assets/drizzle/styles/components/item.css [hunk 1]
```diff
@@ -21,3 +21,8 @@
     width: 1.5em;
   }
 }
+
+.Item-codePreview {
+  max-height: 30em;
+  overflow-y: auto;
+}
```
Comment (saralohr) [inline src/assets/drizzle/styles/components/item.css]: The code samples for the panels were getting really long. This limits their height and adds vertical scroll.

File: src/patterns/components/panel/complex-table.hbs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            </p>
+          </div>
+        </div>
+      {{/each}}
+    </div>
+    <div class="Panel-footer">
+      <button class="Button Button--negative">Sit</button>
+      <button class="Button Button--primary">Amet</button>
+    </div>
+  </div>
```
Comment (captain-pj) [inline src/patterns/components/panel/complex-table.hbs]: Is this one not going to be responsive? It's pretty crunched at small screen size.

Comment (saralohr) [reply src/patterns/components/panel/complex-table.hbs]: Not yet.. the complex tables will likely need solutions more tailored to the content. I made a demo (https://teamsnap-ui-patterns.netlify.com/demos/tables.html) with a few responsive options, but these haven't been brought into the library yet.

File: src/assets/drizzle/styles/components/item.css (hunk 1 above)
Comment (williamdingwall) [reply src/assets/drizzle/styles/components/item.css]: Yeah that's a good call

**Pull request merged**